Source/Core/DolphinQt/NetPlay/NetPlayDialog.cpp

```diff
@@ -71,6 +71,25 @@ NetPlayDialog::NetPlayDialog(QWidget* parent)
   CreateChatLayout();
   CreatePlayersLayout();
   CreateMainLayout();
+
+  int buffer_size = Config::Get(Config::NETPLAY_HOST_BUFFER_SIZE);
```

Comment:
All of this needs to be moved to `CreateMainLayout`, otherwise it will just segfault immediately.